sources/PseudoTerminal.m

```diff
@@ -5637,6 +5637,34 @@ - (IBAction)moveTabRight:(id)sender
     [self tabsDidReorder];
 }
 
+- (IBAction)increaseRow:(id)sender {
+    if (![iTermProfilePreferences boolForKey:KEY_DISABLE_WINDOW_RESIZING inProfile:self.currentSession.profile]) {
```

Comment:
You don't want this if statement--the feature should work even if terminal-initiated resizing is off